src/test/java/seedu/address/logic/parser/AddressBookParserTest.java

```diff
@@ -49,19 +49,39 @@ public void parseCommand_add() throws Exception {
     }
 
     @Test
+    public void parseCommand_add_alias() throws Exception {
```

Comment:
I suggest using `parseCommand_addAlias()` as a change to the naming convention, without the use of two underscores. (Perhaps this would allow you to pass your _codacy/pr_ check too)